app/person-escort-record/controllers/print-record.js

```diff
@@ -0,0 +1,58 @@
+const { filter, orderBy, sortBy } = require('lodash')
+
+const presenters = require('../../../common/presenters')
+
+function printRecord(req, res) {
+  const { framework, personEscortRecord, move } = req
```

Comment:
```js
  const { framework, personEscortRecord, move = {} } = req
  const reference = move.reference
  const {profile = {}} = move
```
utterly trivial comment, but reads nicer to me

Reply:
I've tweaked this a little to support the record being accessed via a move or directly